resolwe/storage/tests/test_management.py

```diff
@@ -0,0 +1,522 @@
+import logging
+from io import StringIO
+
+from django.core.management import call_command
+from django.db.models import Q
+from django.test import TestCase
+
+from resolwe.flow.models import Data, Process
+from resolwe.storage.management.commands.compare_models_and_csv import (
+    FileIterator,
+    ModelIterator,
+    get_etag,
+    get_filename,
+    get_subpath,
+    map_subpath_locations,
+    parseline,
+)
+from resolwe.storage.models import FileStorage, ReferencedPath, StorageLocation
+
+
+class CompareModelsTestCase(TestCase):
+    def setUp(self):
+        self.proc = Process.objects.create(
+            contributor_id=1, name="test.process", slug="test_process"
+        )
+        # subpath 100
+        self.fs100 = FileStorage.objects.create()
+        self.d100 = Data.objects.create(
+            name="test_data100",
+            process_id=self.proc.id,
+            contributor_id=1,
+            location=self.fs100,
+        )
+        self.sl100 = StorageLocation.objects.create(
+            file_storage_id=self.fs100.id, url="100"
+        )
+        # subpath 123
+        self.fs123 = FileStorage.objects.create()
+        self.d123 = Data.objects.create(
+            name="test_data123",
+            process_id=self.proc.id,
+            contributor_id=1,
+            location=self.fs123,
+        )
+        self.sl123 = StorageLocation.objects.create(
+            file_storage_id=self.fs123.id, url="123"
+        )
+
+        self.make_file = lambda name, sl: ReferencedPath.objects.create(
+            path=name, awss3etag="hashhashhash"
+        ).storage_locations.add(sl)
+
+        # all paths present in csv:
+        self.make_file("names/000.json", self.sl100)
+        self.make_file("names/001.json", self.sl100)
+        self.make_file("names/002.json", self.sl100)
+        self.make_file("names/003.json", self.sl100)
+        self.make_file("names/004.json", self.sl100)
+        self.make_file("names/005.json", self.sl100)
+
+        self.make_file("d.json", self.sl123)
+        self.make_file("f.json", self.sl123)
+        self.make_file("j.json", self.sl123)
+        self.make_file("k.json", self.sl123)
+        self.make_file("l.json", self.sl123)
+
+        first_half = "../resolwe/storage/tests"
+        second_half = "/files/compare_models_and_csv.csv"
+        self.csv_filename = first_half + second_half  # 79ch line length limit
```

Comment:
Please see https://github.com/genialis/resolwe/blob/67adcf4eed3813c81c04b825e4f97628c89782a8/resolwe/flow/tests/test_docs.py#L25 for reference how to refer to files.